### PR TITLE
[WiP] ARGO-1549 New helper tool that can replay avro data on AMS with customizable datestamp

### PR DESCRIPTION
**Pull request by kzailac**

### CODE REVIEW

File: bin/replay-avro-data.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+        return True, data
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description="Helper tool that can replay avro data on AMS with "
+                    "customizable datestamp"
+    )
+    parser.add_argument("-h", dest='host', nargs=1, metavar="ams_host",
```
Comment (vrdel) [inline bin/replay-avro-data.py]: `-h` is reserved argument of `ArgumentParser` module:
`argparse.ArgumentError: argument -h/--help: conflicting option string(s): -h`

Have it `-a`

Comment (kzailac) [reply bin/replay-avro-data.py]: Fixed as you asked for.

File: bin/replay-avro-data.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        print "Error: " + data
+        raise SystemExit(1)
+
+    amshost = args.host  # ams host
+    amsproject = args.project  # ams project
+    amstoken = args.token  # ams token
+    amstopic = args.topic  # ams topic
+    amsreport = args.report  # report - dirname from customer.conf is used for this one
+    amsbulk = 100
+    amspacksinglemsg = True
```
Comment (vrdel) [inline bin/replay-avro-data.py]: Have this one as string 'True' and it will be fine.

Comment (kzailac) [reply bin/replay-avro-data.py]: Done.

**Pull request merged**